multiraft/multiraft.go

```diff
@@ -982,6 +985,15 @@ func (s *state) handleWriteReady(readyGroups map[uint64]raft.Ready) {
 		g.writing = true
 
 		gwr := &groupWriteRequest{}
+		var err error
+		gwr.replicaID, err = s.Storage.ReplicaIDForStore(roachpb.RangeID(groupID), s.storeID)
+		if err != nil {
+			if log.V(1) {
+				log.Warningf("failed to look up replica ID for range %v (disabling replica ID check): %s",
+					groupID, err)
+			}
+			gwr.replicaID = 0
```

Comment:
I assume you care about the 0 value. How about creating a const illegalReplicaID = 0